api/v1beta2/ibmvpcmachine_types.go

```diff
@@ -78,6 +78,14 @@ type IBMVPCMachineSpec struct {
 	// SSHKeys is the SSH pub keys that will be used to access VM.
 	// ID will take higher precedence over Name if both specified.
 	SSHKeys []*IBMVPCResourceReference `json:"sshKeys,omitempty"`
+
+	// AdditionalVolumes is the list of additional volumes attached to the instance
```

Comment:
```suggestion
	// additionalVolumes is the list of additional volumes attached to the instance
```
We are supposed to use the user facing name in comment so user will find it easy when we use go doc.